src/index.js

```diff
@@ -18,11 +18,8 @@ import { config } from './utils/oktaConfig';
 import Signup from './components/pages/RoleSignup/Signup';
 import Mentee from './components/pages/RoleSignup/Applications/Mentee';
 import Mentor from './components/pages/RoleSignup/Applications/Mentor';
-
-import AppError from './components/pages/RoleSignup/Applications/AppError';
+import AppSuccess from './components/pages/RoleSignup/Applications/AppSuccess';
```

Comment:
Thank you for finally changing this!! I dont know why it is changed in here though... Does it have anything to do with viewing all meetings directly? 
If not `git add <file_name>` might help add only desired files to your PR. Or you can use the GUI in VSCode

Reply:
What are you referring to?